test/artefacts/marriage-abroad/belarus/ceremony_country/partner_local/opposite_sex.txt

```diff
@@ -40,7 +40,6 @@ You must take the form with you to your appointment.
 
 Service | Fee
 -|-
-Administering an oath or making a declaration | £55
```

Comment:
See comments and screenshots [here](https://github.com/alphagov/smart-answers/pull/2413/files#r57885971)